diff --git a/cfunits/units.py b/cfunits/units.py
index caac22f..01b931b 100644
--- a/cfunits/units.py
+++ b/cfunits/units.py
@@ -79,7 +79,13 @@ _ut_read_xml.argtypes = (_c_char_p, )
 _ut_read_xml.restype = _c_void_p
 
 #print('units: before _udunits.ut_read_xml(',_unit_database,')')
-_ut_system = _ut_read_xml(None)
+if sys.platform == 'darwin' or sys.platform.startswith("linux"):
+    path_to_xml = None  # Use the default path
+else:
+    path_to_xml = _c_char_p(
+        os.path.join(sys.prefix, "Lib/site-packages/cfunits/etc/udunits/udunits2.xml")
+    )
+_ut_system = _ut_read_xml(path_to_xml)
 #print('units: after  _udunits.ut_read_xml(',_unit_database,')')
 
 # Reinstate the reporting of error messages
